packages/blade/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @razorpay/blade
 
+## 6.6.1
+
+### Patch Changes
+
+- fe89e6f6: fix: tree-shaking in blade components
+- 7817c9e3: feat(Box): add different types for `display` on react native
+- c6512ba0: fix(Alert, Card): set `box-sizing` as `border-box` for Alert and Card
```

Comment:
Should we do minor? All 3 of them are not very atomic changes, quite substantial changes imo.

Reply:
It won't have any sort of impact for the end-user, right? Even tree shaking will have an impact only in certain specific cases.